modules/adnuntiusBidAdapter.js

```diff
@@ -0,0 +1,103 @@
+// import * as utils from 'src/utils';
+// import { config } from 'src/config';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+const BIDDER_CODE = 'adnuntius';
+const ENDPOINT_URL = 'https://delivery.adnuntius.com/i?tzo=-60&format=json';
+
+const getTopWindowReferrer = () => {
+  try {
+    return window.top.document.referrer;
+  } catch (e) {
+    return '';
+  }
+}
+
+export const spec = {
+  code: BIDDER_CODE,
+
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.auId || (bid.params.member && bid.params.invCode));
+  },
+
+  buildRequests: function (validBidRequests) {
+    const networks = {}
+    const bidRequests = {}
+    const requests = []
+
+    for (const bid of validBidRequests) {
+      const network = bid.params.network
+      bidRequests[network] = bidRequests[network] || []
+      bidRequests[network].push(bid)
+
+      networks[network] = networks[network] || {}
+      networks[network].adUnits = networks[network].adUnits || []
+      networks[network].adUnits.push({ ...bid.params.targeting, auId: bid.params.auId })
```

Comment:
This is more of a style-related remark: did you deliberately omit `;` from end of lines, because I see that you're using it somewhere, while on other places you're not?

Reply:
Sorry just too used with not using them. will add. :)